### PR TITLE
Accept common URLs as repo spec in parse_github_repo_spec(); closes #90

### LINKED ISSUE #90: install_github() could accept full URL to GitHub repository
<a href="https://github.com/kevinushey"></img></a> **Issue by [kevinushey](https://github.com/kevinushey)**
_Thursday Sep 01, 2016 at 15:45 GMT_
_Originally opened as https://github.com/hadley/devtools/issues/1324_

----

Sometimes, when I am exceedingly lazy, I just try to copy + paste the path to a GitHub repository into `install_github`. It would be 🔥🔥🔥 if `install_github` were able to magically parse that.

### PR DESCRIPTION
**Pull request by jennybc**

### CODE REVIEW

**Comment by gaborcsardi** [comment]
Thanks! Looks pretty good! I have wanted this for a long time!

File: NEWS.md [hunk 1]
```diff
@@ -1,6 +1,9 @@
 
 # 1.1.0
 
+* Accept browser, HTTPS, or SSH URL in GitHub repo specification,
```
Comment (gaborcsardi) [inline NEWS.md]: CAn you please move this up to a new section. Version 1.1.0.9000? Thanks!

Comment (gaborcsardi) [reply NEWS.md]: 1.1.0 is actually the current version....

**Comment by gaborcsardi** [comment]
Perfect. Thanks much again!

File: NEWS.md (hunk 1 above)
Comment (jennybc) [reply NEWS.md]: ok done!

**Pull request merged**